### DISCUSSION TITLE
Validar Selected no Po-Table

### DISCUSSION THREAD

**Discussion by joaomauriciotecadi** [po-ui/po-angular#2114] [Tire suas dúvidas]
Olá!

Sabem se existe alguma forma de poder validar se é possivel ou não selecionar uma linha no po-table?

**Comment by jpinto10** [comment]
Estou com a mesma necessidade!

**Comment by alinelariguet** [comment]
Bom dia! @joaomauriciotecadi e @jpinto10 já existe no po-table os eventos que são emitidos durante a seleção e também existem métodos para selecionar ou desselecionar uma linha específica, acredito que isso solucione a necessidade de vocês, evento para ser notificado da seleção, validação a partir do evento e desseleção se necessário via método, pode ser exibido um dialog para dar um feedback para o usuário caso ele não possa selecionar aquela linha.
Att.

**Comment by joaomauriciotecadi** [comment]
Boa tarde @alinelariguet , realmente eu verifiquei melhor o componente e conseguir realizar uma validação. Inclusive, @jpinto10 segue um repo com exemplo: https://github.com/joaomauriciotecadi/poui-teste-table
